src/components/Popups/PopupWithForm/PopupWithForm.js

```diff
@@ -28,8 +31,11 @@ const PopupWithForm = (props) => {
         onClick={props.closePopup}
       ></button>
       <form
+        name="popupform"
```

Comment:
`        name="popupform"
`
Attribute `name` should be unique in any tag `form` to provide built-in form handling